### DISCUSSION TITLE
No public access to assets (except thumbnails) when using custom storages

### DISCUSSION THREAD

**Discussion by mwisniewskilm** [pimcore/pimcore#11179] [General]
I am using Pimcore version 10.0.8, standard apache docker, and i configured storages for my assets, here's my configuration:
[storages.txt](https://github.com/pimcore/pimcore/files/7861716/storages.txt)

And here's the problem: 
Files are landing in correct locations, when asset is acessed or downloaded from admin: it'works perfectly, when it's acessed through website by thumbnail, it works too, but when im using public link to access file directly im getting 'No route found' exception.
My friend also reproduced this on nginx, can anyone help to find a solution ? Is that a bug, or wrong configuration?

**Comment by dvesh3** [comment]
You have to set frontend prefixes when using custom storage for assets https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Setup_and_Hosting/File_Storage_Setup.html#page_File-Storage-Setup
